docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,181 @@
+  - if the draw_pattern is given, get the draw pattern: `if shw_eq.draw_pattern != null: draw_pattern = shw_eq.draw_pattern`
+  - otherwise, determine the draw pattern based off the first_hour_rating of the shw_eq: `else:`
+    - First get the first hour rating (gallons): `first_hour_rating = shw_eq.first_hour_rating`
+    - if the first_hour_rating is less than 18 gallons, draw_pattern is "VERY SMALL": `if first_hour_rating < 18: draw_pattern = "VERY_SMALL"`
+    - otherwise if the first_hour_rating is less than 51 gallons, draw pattern is LOW: `elif first_hour_rating < 51: draw_pattern = "LOW"`
+    - otherwise if the first_hour_rating is less than 75 gallons, draw pattern is MEDIUM: `elif first_hour_rating < 75: draw_pattern = "MEDIUM"`
+    - otherwise draw pattern is HIGH: `else: draw_pattern = "HIGH"`
+  - get the capacity of the SHW system (make sure to convert to btu/hr): `input_capacity = swh_eq.input_power`
+  - get the volume of the tank (make sure to convert to gallons): `storage_volume = swh_eq.distribution_system.tank.storage_capacity`
+  - calculate the capacity_per_volume based on the input_capacity and the storage_volume: `capacity_per_volume = shw_eq.rated_capacity / storage_volume`
+  - get the efficiency metric values: `efficiency_metric_values = swh_eq.efficiency_metric_values`
+  - get the efficiency metric types: `efficiency_metric_types = swh_eq.efficiency_metric_types`
+  - create a dictionary of the efficiency data: `efficiency_metric_dict = dict(zip(efficiency_metric_types, efficiency_metric_values))
+  - get the thermal efficiency: `thermal_efficiency = efficiency_metric_dict["THERMAL_EFFICIENCY"]`
+  - get the uniform energy factor: `uniform_energy_factor = efficiency_metric_dict["UNIFORM_ENERGY_FACTOR"]`
+  - what follows is logic that implements Table 7.8 - this table has multiple criteria, so I think it's simpler to implement with if / else logic than a table lookup
+  - if the swh_type is OTHER, set rule_note to "Water heater type is not a recognized type", and continue to rule assertion: `if swh_type == "OTHER": rule_note = "Water heater type is not a recognized type"; GO TO RULE ASSERTION`
+  - create a boolean has_fault and set it to false: `has_fault = false`
+  - if the water heater is electric resistance, or storage and the capacity is <= 12kW (40945 btu/hr), look at Appendix F: `if swh_type in ["ELECTRIC_RESISTANCE_STORAGE"] and input_capacity <= 40945.7:`
+    - check the size of the storage tank.  If it's less than 2 gallons, provide a note, and continue to rule assertion: `if storage_volume < 2: rule_note = "The storage volume is not consistent with an electric storage water heater as it is less than 20 gallons."; GO TO RULE ASSERTION`
```

Comment:
Recommended note to provide the reviewer with the full context:  "The storage tank volume is less than 2 gallons and based on the size categories in ASHRAE 90.1 Table 7.8 and Appendix F it would not be classified as an electric storage water heater. Consequently, this rule was not assessed for this service water heater."